webapp/src/main/java/com/box/l10n/mojito/monitoring/QuartzPendingJobsReportingTask.java

```diff
@@ -0,0 +1,69 @@
+package com.box.l10n.mojito.monitoring;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+import javax.sql.DataSource;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tags;
+
+@Component
+@ConditionalOnProperty(value = {"l10n.management.metrics.quartz-metrics.enabled",
```

Comment:
aren't those two monitoring system independent? since it is opt-in, should it just check a single flag?

Reply:
Done